etc/docker/elyra_development/README.md

```diff
@@ -0,0 +1,37 @@
+<!--
+{% comment %}
+Copyright 2018-2022 Elyra Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+### Elyra Development Docker Image
+
+Builds an Elyra development workspace image. This image provides all the necessary libraries and frameworks to build and test
+Elyra changes. 
+
+#### Building the image
+
+To build a custom version of this container image:
+1. Clone this repository.
```

Comment:
It might be helpful to include an example `git clone` command.  It would be great if any referenced paths corresponded to the example paths below.

It seems "self-documenting" paths are common these days.  E.g., /path/to/elyra/root.